reference/constraints/Bic.rst

```diff
@@ -0,0 +1,107 @@
+Bic
+===
+
+.. versionadded:: 2.8
+    The Bic constraint was introduced in Symfony 2.8.
+
+This constraint is used to ensure that a value has the proper format of an
+`Business Identifier Code (BIC)`. BIC is an internationally agreed means to
```

Comment:
the closing `` ` `` needs to be followed by an underscore: `` `Business Identifier Code (BIC)`_``